WordPress/Classes/ViewRelated/Support/SupportChatBot/support_chat_widget.css

```diff
@@ -3,3 +3,14 @@
 a.floating-button {
     display: none !important;
 }
+
+.docsbot-wrapper {
+    width: 100% !important;
+    height: 100% !important;
+    position: fixed !important;
+    top: 0 !important;
+    left: 0 !important;
+    right: 0 !important;
+    bottom: 0 !important;
+    max-height: none !important;
+}
```

Comment:
I tested this and it works well. 

I'll leave one observation though (which isn't a blocker).

The WKWebView [adds a default 8px margin](https://stackoverflow.com/a/34276426/1305067), so that can be turned off by setting the margin to zero on the body:
```
body {
    margin: 0;
}
```
However, this needs to be imported into the main HTML page (not the shadow DOM), so it can't go in `support_chat_widget.css`. Instead, it would need to go in a separate stylesheet such as `support_chat_widget_page.css` and be imported into `support_chat_widget.html` e.g.:
```
<html>
<head>
    <meta meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <link rel="stylesheet" type="text/css" href="support_chat_widget_page.css" />
...
```

---

Removing the default margin means the chatbot UI fills to the edges of the screen and we can change the positioning to the default and reset the width and height:

```
.docsbot-wrapper {
    position: static;
    width: auto;
    height: auto;
}
```
This resets the styles back to the defaults, which seems to produce the desired effect (there's existing DocsBot CSS that gives this a height, but I'm not sure of the details of that).

---

While `!important` isn't often the right choice, I can see why it's a viable option here since we want to have the final say on the styling (and we have no control over the CSS from DocsBot or how it'll change in the future). So I think your current solution is good, and my observations above leave open the possibility of DocsBot overriding CSS by using more specific selectors than ours.

Reply:
> The WKWebView [adds a default 8px margin](https://stackoverflow.com/a/34276426/1305067)

I didn't know that, thanks!

> Removing the default margin means the chatbot UI fills to the edges of the screen and we can change the positioning to the default and reset the width and height

I tried it out, it works well! 👍

> While !important isn't often the right choice, I can see why it's a viable option here since we want to have the final say on the styling

Yes. We're not creating a long-term solution here and we're aware of that. Adding `!important` limits the choice of granular improvements within the chat bot breaking our UI, although we have no defense against more drastic improvements.